src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/FilteredTokens.java

```diff
@@ -35,9 +38,21 @@ public String getFilteredTokens() {
         return filteredTokens;
     }
 
+    private static ArrayList<String> tokenToString(ArrayList<Token> tokens) {
+        ArrayList<String> tokenStrings = new ArrayList<>();
+        for(Token t : tokens) {
+            tokenStrings.add(Objects.toString(t, null));
+            if(originalId == -1) {
+                originalId = t.getOriginID();
+            } else continue;
+        }
+        return tokenStrings;
+    }
+
     ArrayList<Token> filterTokens(ArrayList<Token> tokens) {
+        tokenToString(tokens);
         //tokens.set(classstopwords.methodprovided(tokens));
-        //tokens.set(classirr.methodprovided(tokens));
+        //tokens.set(classirr.searchForIrregularWords(tokens));
         //tokens.set(classstem.methodprovided(tokens));
```

Comment:
`classstem.methodprovided(tokens)` can be replaced with: `new Stemmer().stem(tokens)`

Reply:
done